### PR TITLE
[permissions] Rename labels in the front end

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
These were copied over directly from Leigh's suggestions so there a few things to be mindful of:
- some typos were fixed as I was entering them but I might have missed some too
- I dont necessarily agree with all the own sites/all sites combinations so I will be putting comments on a line by line basis
- some permissions were not renamed because they were not in Leigh's list so feel free to suggest a name (anyone)
- some permissions were deleted because they are not used in the codebase any longer

I have not yet made the RB changes nor the patch that goes with it until we finalize the renaming.

### CODE REVIEW

**Comment by ridz1208** [comment]
@driusan as a general note do you think we should remove all the `module` words ? seems redundant ?

File: SQL/0000-00-01-Permission.sql [hunk 1]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
 
 INSERT INTO `permissions` VALUES
     (1,'superuser','Superuser - supersedes all permissions','1'),
-    (2,'user_accounts','User management','2'),
-    (3,'user_accounts_multisite','Across all sites create and edit users','2'),
-    (4,'context_help','Edit help documentation','2'),
+    (2,'user_accounts','User Accounts Module View Own Site','2'),
+    (3,'user_accounts_multisite','User Accounts Module create and edit users','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:54]:
```suggestion
    (3,'user_accounts_multisite','User Accounts Module create and edit users All Sites','2'),
```

File: SQL/0000-00-01-Permission.sql [hunk 2]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (4,'context_help','Help Module Edit Documentation','2'),
     (5,'bvl_feedback','Behavioural QC','1'),
-    (6,'imaging_browser_qc','Edit imaging browser QC status','2'),
-    (7,'mri_efax','Edit MRI Efax files','2'),
-    (8,'send_to_dcc','Send to DCC','2'),
-    (9,'unsend_to_dcc','Reverse Send from DCC','2'),
-    (10,'access_all_profiles','Across all sites access candidate profiles','2'),
+    (6,'imaging_browser_qc','Imaging Browser Module Edit QC Status','2'),
+    (8,'send_to_dcc','Candidate Profile Module Send to DCC All Sites','2'),
+    (9,'unsend_to_dcc','Candidate Profile Module Reverse Send from DCC own Site','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:59]: why is one `All sites` and the other `Own site` is that actually the case ?

Comment (ridz1208) [reply SQL/0000-00-01-Permission.sql:59]: Also the module is technically instrument_list not candidate_profile

File: SQL/0000-00-01-Permission.sql [hunk 3]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
-    (37,'server_processes_manager','View and manage server processes','2'),
-    (38,'imaging_uploader','Imaging Uploader','2'),
-    (39,'acknowledgements_view','View Acknowledgements','2'),
-    (40,'acknowledgements_edit','Edit Acknowledgements','2'),
-    (41,'dataquery_view','View Data Query Tool','2'),
-    (42,'genomic_data_manager','Manage the genomic files','2'),
-    (43,'media_write','Media files: Uploading/Downloading/Editing','2'),
+    (12,'examiner_view','Examiner Module Edit Own Site','2'),
+    (13,'examiner_multisite','Examiner Module Edit All Sites','2'),
+    (15,'timepoint_flag','Candidate Parameter Module Edit Exclusion Flags Own Site','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:64]: This permission does not seem to be used anywhere

@driusan confirm ?

File: SQL/0000-00-01-Permission.sql [hunk 4]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (17,'conflict_resolver','Conflict Resolver Module Edit Own Site','2'),
+    (18,'data_dict_view','Data Dictionary Module View','2'),
+    (19,'violated_scans_view_allsites','MRI Violated Scan Module View All Sites','2'),
+    (22,'config','Configuration Settings Edit','2'),
+    (23,'imaging_browser_view_site','Imaging Browser Module View Own Site','2'),
+    (24,'imaging_browser_view_allsites','View all-sites Imaging Browser pages','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:70]:
```suggestion
    (24,'imaging_browser_view_allsites','Imaging Browser Module View All Sites','2'),
```

File: SQL/0000-00-01-Permission.sql [hunk 5]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (12,'examiner_view','Examiner Module Edit Own Site','2'),
+    (13,'examiner_multisite','Examiner Module Edit All Sites','2'),
+    (15,'timepoint_flag','Candidate Parameter Module Edit Exclusion Flags Own Site','2'),
+    (17,'conflict_resolver','Conflict Resolver Module Edit Own Site','2'),
+    (18,'data_dict_view','Data Dictionary Module View','2'),
+    (19,'violated_scans_view_allsites','MRI Violated Scan Module View All Sites','2'),
+    (22,'config','Configuration Settings Edit','2'),
+    (23,'imaging_browser_view_site','Imaging Browser Module View Own Site','2'),
+    (24,'imaging_browser_view_allsites','Imaging Browser Module View All Sites','2'),
+    (25,'dicom_archive_view_allsites','DICOM Module Archive Access','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:71]:
```suggestion
    (25,'dicom_archive_view_allsites','DICOM Archive Module Archive Access','2'),
```

??

File: SQL/0000-00-01-Permission.sql [hunk 6]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (28,'instrument_builder','Instrument Builder Module Edit Instruments','2'),
+    (29,'data_dict_edit','Data Dictionary Edit','2'),
+    (31,'candidate_parameter_view','Candidate Parameter Module View Own Site','2'),
+    (32,'candidate_parameter_edit','Candidate Parameter Module Edit Own Site','2'),
+    (33,'genomic_browser_view_site','Genomic Browser Module View Own Sites','2'),
+    (34,'genomic_browser_view_allsites','Genomic Browser Module View All Sites','2'),
+    (35,'document_repository_view','Document Repository Module View and Upload Own File','2'),
+    (36,'document_repository_delete','Document Repository: Delete All Files','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:79]: not sure if the `own file`/`all files` make any sens here

Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:79]:
```suggestion
    (35,'document_repository_view','Document Repository Module View and Upload Own File','2'),
    (36,'document_repository_delete','Document Repository Module Delete All Files','2'),
```

File: SQL/0000-00-01-Permission.sql [hunk 7]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (37,'server_processes_manager','Server Process Module View','2'),
+    (38,'imaging_uploader','Imaging Uploader Module Upload Images for Own Site','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:81]:
```suggestion
    (38,'imaging_uploader','Imaging Uploader Module Upload Images Own Site','2'),
```

for consistency

File: SQL/0000-00-01-Permission.sql [hunk 8]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (39,'acknowledgements_view','Acknowledgement Module View All Sites','2'),
+    (40,'acknowledgements_edit','Acknowledgement Module Edit All Sites','2'),
+    (41,'dataquery_view','Data Query Tool View All Sites','2'),
+    (42,'genomic_data_manager','Genomic Browser Module Edit Genomic Files at Own Site','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:85]:
```suggestion
    (42,'genomic_data_manager','Genomic Browser Module Edit Genomic Files Own Site','2'),
```

for consistency

File: SQL/0000-00-01-Permission.sql [hunk 9]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (43,'media_write','Media Module Upload, Download, Edit All Sites','2'),
     (44,'media_read','Media files: Browsing','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:87]:
```suggestion
    (44,'media_read','Media Module View File names own site','2'),
```

File: SQL/0000-00-01-Permission.sql [hunk 10]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (35,'document_repository_view','Document Repository Module View and Upload Own File','2'),
+    (36,'document_repository_delete','Document Repository Module Delete All Files','2'),
+    (37,'server_processes_manager','Server Process Module View','2'),
+    (38,'imaging_uploader','Imaging Uploader Module Upload Images for Own Site','2'),
+    (39,'acknowledgements_view','Acknowledgement Module View All Sites','2'),
+    (40,'acknowledgements_edit','Acknowledgement Module Edit All Sites','2'),
+    (41,'dataquery_view','Data Query Tool View All Sites','2'),
+    (42,'genomic_data_manager','Genomic Browser Module Edit Genomic Files at Own Site','2'),
+    (43,'media_write','Media Module Upload, Download, Edit All Sites','2'),
+    (44,'media_read','Media Module View File names own site','2'),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:87]: these dont seem consistent with the rest. also not sure if you actually need write for being able to download

File: SQL/0000-00-01-Permission.sql [hunk 11]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (45,'issue_tracker_reporter','Issue Tracker Module Edit Own, Comment on All', 2),
+    (46,'issue_tracker_developer','Issue Tracker Module Edit All', 2),
+    (47,'imaging_browser_phantom_allsites','Imaging Browser Module Access Phantom Data All Sites', 2),
+    (48,'imaging_browser_phantom_ownsite','Can access only phantom data from own site in Imaging Browser', 2),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:91]:
```suggestion
    (48,'imaging_browser_phantom_ownsite','Imaging Browser Module Access Phantom Data Own Site', 2),
```

File: SQL/0000-00-01-Permission.sql [hunk 12]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (49,'data_release_view','Data Rlease Module View', 2),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:92]:
```suggestion
    (49,'data_release_view','Data Release Module View', 2),
```

File: SQL/0000-00-01-Permission.sql [hunk 13]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (41,'dataquery_view','Data Query Tool View All Sites','2'),
+    (42,'genomic_data_manager','Genomic Browser Module Edit Genomic Files at Own Site','2'),
+    (43,'media_write','Media Module Upload, Download, Edit All Sites','2'),
+    (44,'media_read','Media Module View File names own site','2'),
+    (45,'issue_tracker_reporter','Issue Tracker Module Edit Own, Comment on All', 2),
+    (46,'issue_tracker_developer','Issue Tracker Module Edit All', 2),
+    (47,'imaging_browser_phantom_allsites','Imaging Browser Module Access Phantom Data All Sites', 2),
+    (48,'imaging_browser_phantom_ownsite','Imaging Browser Module Access Phantom Data Own Site', 2),
+    (49,'data_release_view','Data Release Module View', 2),
+    (50,'data_release_upload','Data Release: Upload file', 2),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql:93]:
```suggestion
    (50,'data_release_upload','Data Release Module Upload', 2),
```

File: SQL/0000-00-01-Permission.sql [hunk 14]
```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
...
+    (51,'data_release_edit_file_access','Data Release Module: Grant View Permission', 2),
```
Comment (ridz1208) [inline SQL/0000-00-01-Permission.sql]: i think these 3 should have `all sites` at the end

File: SQL/0000-00-01-Permission.sql (hunk 2 above)
Comment (driusan) [reply SQL/0000-00-01-Permission.sql:59]: I don't know

File: SQL/0000-00-01-Permission.sql (hunk 3 above)
Comment (driusan) [reply SQL/0000-00-01-Permission.sql:64]: Yes, I think that was an NIHPD thing.

**Comment by driusan** [comment]
I agree with not needing the word "Module". The convention that we (partially) have now of it just being "Modulename:" (with a colon) seems clearer and easier to read.

**Pull request closed without merging**